data/human/free worlds 4 epilogue.txt

```diff
@@ -32,7 +32,7 @@ mission "FW Epilogue: Ijs and Katya"
 					goto excitement
 				`	"It's kind of hard to adjust to civilian life."`
 					goto adjust
-				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been implemented yet."`
+				`	"I want to explore the rest of the galaxy, but apparently not all of it has been implemented yet."`
```

Comment:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all of the game has been implemented yet."`
```

Whole thing is still a 4th wall break, so we may as well leave the direct reference here.

Reply:
I was considering this, but it seemed odd since the original was saying "that part of the game", when here, it's "the game", which are two different meanings.